Makefile

```diff
@@ -83,16 +74,26 @@ swift_snapshot_install:
 	curl https://swift.org/builds/development/xcode/$(SWIFT_SNAPSHOT)/$(SWIFT_SNAPSHOT)-osx.pkg -o swift.pkg
 	sudo installer -pkg swift.pkg -target /
 
+spm_bootstrap: spm_teardown
+	ln -s /Library/Developer/Toolchains/$(SWIFT_SNAPSHOT).xctoolchain/usr/local/include/clang-c /usr/local/include/clang-c
+	sudo cp Source/Clang_C/module.modulemap /usr/local/include/clang-c/module.modulemap
+	ln -s /Library/Developer/Toolchains/$(SWIFT_SNAPSHOT).xctoolchain/usr/lib/libclang.dylib /usr/local/lib/libclang.dylib
+	mkdir -p /usr/local/include/sourcekitdInProc
+	ln -s /Library/Developer/Toolchains/$(SWIFT_SNAPSHOT).xctoolchain/usr/lib/sourcekitd.framework/Headers/sourcekitd.h /usr/local/include/sourcekitdInProc/sourcekitd.h
+	curl https://static.realm.io/libsourcekitdInProc/$(SWIFT_SNAPSHOT)/libsourcekitdInProc.dylib -o /usr/local/lib/libsourcekitdInProc.dylib
+
+spm_teardown:
+	rm -rf /usr/local/include/clang-c /usr/local/lib/libclang.dylib
```

Comment:
Since the files in the toolchain don't change, I think I'll avoid the use of `sudo` here and copy the files instead. Then `spm_teardown` will remove the module map when deleting `clang-c`.